instagram/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import './App.css';
+import PostsPage from './components/PostContainer/PostsPage';
+import Authenticate from './Authentication/Authenticate';
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state=({
+      users: [1]
+    })
+  }
+
+  login = () => {
+    localStorage.setItem('username', 'anonymous') 
+    this.setState();
```

Comment:
It looks like you have some artifacts here that you aren't using anymore. Why not get rid of what you don't need? At this point, it looks like `App` could be a functional component, or perhaps could just be removed completely and then have `PostsPage` take its place (still with the `Authenticate` HOC).